packages/block-library/src/template-part/edit/index.js

```diff
@@ -23,7 +29,7 @@ import TemplatePartPlaceholder from './placeholder';
 import TemplatePartSelection from './selection';
 
 export default function TemplatePartEdit( {
-	attributes: { postId: _postId, slug, theme, tagName: TagName = 'div' },
+	attributes: { postId: _postId, slug, theme, tagName: TagName = 'section' },
```

Comment:
If the default is div, shouldn't this remain div, not section?

Reply:
Changed default to `div` :+1: